src/routes/countryRoutes.js

```diff
@@ -0,0 +1,22 @@
+import Router from "express";
+import controllers from "../controllers";
+import Authentication from "../middlewares/authenticate";
+
+const { verifyAdmin, verifyToken } = Authentication;
+
+const { countriesController } = controllers;
+const {
+  listCountries, getCountry, deleteCountry, updateCountry
+} = countriesController;
+
+const router = Router();
+
+router.get("/get-country", getCountry);
```

Comment:
since they are admin routes, let your routes be preceded with admin e.g instead of `router.get("/get-country", getCountry);` let it be => `router.get("/admin/get-country", getCountry);`

Also, delete this your routes file and add your code to fiyin's routes file. rename fiyin's own as `countryRoutes`